### ISSUE TITLE
DocumentStatus.md ideas

### ISSUE THREAD

**Issue by podborski** [MPEGGroup/FileFormat#62]
I find the page very very useful. Especially one can use the automation script to automatically download the latest versions of the important documents so that you always have the most up to date documents without looking for them in the MDMS.

One idea maybe on how to improve it.

Currently `DocumentStatus.md` is not really easy to edit (or parse). Maybe we could have a simple `.csv` file which you could load in the Excel or Numbers for editing. And it would it make much easier to write a document fetcher script to download all the documents. I believe `.csv` can also be directly rendered on GitHub, but if needed, a markdown translator should also be very easy to develop.

Just a thought.

**Comment by podborski** [comment]
Yep .csv is rendered on GitHub as shown in this example https://github.com/MPEGGroup/FileFormat/blob/document_status/DocumentStatus.csv

**Comment by podborski** [comment]
@cconcolato I experimented a bit with the script. Can you check https://github.com/MPEGGroup/FileFormat/blob/document_status/DocumentStatus.md and confirm that I din't miss anything? This is created from the CSV file above with the automation script. You can try it yourself:

Generate report in DocumentStatus.md file:
```shell
python fileformat.py -r -i path_to_csv
```

Fetch all output docs:
```shell
python fileformat.py -f -i path_to_csv  -z -o destination_dir
```

**Comment by cconcolato** [comment]
Thanks. I will double-check. This would be very useful and save me a lot of time! 

The thing that I liked with the Markdown is the separation of the specs (1 table per spec), the table of contents, and the removal of the redundancy (number, name, ...) but that could be implemented very easily in JS.

**Comment by edrthomas** [comment]
With an API, could get up-to-date info of the status yes. Expected publication date is unlikely.
